### PR TITLE
chore(v2): update Infima to alpha 26

### LINKED ISSUE #3820: Top nav could be pushed to the next row in some screen width
## 🐛 Bug Report

If the top nav bar has too many items / too much text, it could be pushed to the next row on tablet / old desktop screen that is around 1024px wide, which is wider than the threshold the top nav bar is shrunk into a hamburger button.

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?
Yes

## Real-life Demo

Width: 1024px

https://v2.docusaurus.io/docs/

docusaurus

https://reactnative.dev/docs/getting-started

react-native

https://react-redux.js.org/introduction/quick-start

react-redux

https://redux.js.org/introduction/getting-started

redux

(Good for Redux, it looks like they fixed it. However, there is a minor problem: the viewport is not fit and is wider than the screen, showing the horizontal scrollbar)

## Comment

Maybe get the the top bar's real width after it is loaded, then make some calculation and change the threshold screen width the top nav is shrunk?

## Your Environment

- Docusaurus version used: V2
- Environment name and version: Chrome 86 and Firefox 82
- Operating system and version: Windows 10 Desktop

### LINKED ISSUE #3937: Incorrect navbar style with light mode and dark navbar
## 🐛 Bug Report

I noticed some styling issues when using the dark navbar with the light mode. I don't know if some are on purpose.

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes.

## To Reproduce

Use this configuration in `docusaurus.config.js`:
```js
module.exports = {
  // ...
  themeConfig: {
    // ...
    colorMode: {
      defaultMode: 'light',
      disableSwitch: true,
      respectPrefersColorScheme: false,
    },
    navbar: {
      // ...
      style: 'dark',
    }
  }
}
```

## Expected / Actual behaviors

**Issue 1: navbar background color**
The navbar background color is not the same one that is being used in the dark mode. Is it on purpose?

_Expected:_

Capture d’écran 2020-12-16 à 16 22 19
Expected color to be `#242526` like `--ifm-background-surface-color` in dark mode.

_Actual:_

Capture d’écran 2020-12-16 à 16 21 56

Capture d’écran 2020-12-18 à 09 57 02

_Current workaround in global CSS file:_
```scss
.navbar--dark {
  --ifm-navbar-background-color: #242526;
}
```

**Issue 2: bad link colors**

Some links are not colored in white when browser the site with a small screen (mobile mode): the menu button and the version dropdown.

Capture d’écran 2020-12-16 à 16 21 56
Capture d’écran 2020-12-16 à 16 22 04

## Your Environment

- Docusaurus version used: 2.0.0-alpha.70 (same issues with alpha.69)
- Environment name and version: Firefox 83.0, Node.js v12.19.1
- Operating system and version: Desktop

## Reproducible Demo

Repo and directory: https://github.com/FoalTS/foal/tree/master/docs
Online website: https://foalts.org

### PR DESCRIPTION
**Pull request by lex111**
## Motivation

Resolves #3820 (in most cases this will be enough to fix this since we have truncated the navbar title and reduced spacing between navbar items).

Fixes #3937

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Preview

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

File: packages/docusaurus-theme-classic/src/theme/SkipToContent/styles.module.css [hunk 1]
```diff
@@ -6,9 +6,6 @@
   padding: calc(var(--ifm-global-spacing) / 2) var(--ifm-global-spacing);
   color: var(--ifm-color-emphasis-900);
   background-color: var(--ifm-background-surface-color);
-  border-radius: var(--ifm-global-radius);
```
Comment (lex111) [inline packages/docusaurus-theme-classic/src/theme/SkipToContent/styles.module.css:9]: This is not directly related to updating Infima, just removing unnecessary styles and adding shadow to the skip link (see above).

**Comment by slorber** [comment]
👍
